testsuite/features/qam/reposync/srv_sync_qam_products.feature

```diff
@@ -119,10 +119,18 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I click the Add Product button
     And I wait until I see "RHEL or SLES ES or CentOS 8 Base x86_64" product has been added
 
-  Scenario: SUSE Manager Proxy 4.1 x86_64
+  Scenario: SUSE Manager Proxy 4.2 x86_64
     Given I am on the Products page
-    And I enter "SUSE Manager Proxy 4.1 x86_64" as the filtered product description
-    And I select "SUSE Manager Proxy 4.1 x86_64" as a product
-    Then I should see the "SUSE Manager Proxy 4.1 x86_64" selected
+    When I enter "SUSE Manager Proxy 4.2 x86_64" as the filtered product description
```

Comment:
👍

Reply:
Yeah, it's a bit of anticipating the future, I know...